src/main/java/org/jabref/gui/preferences/file/FileTab.fxml

```diff
@@ -24,12 +24,12 @@
         <Label text="%Do not wrap the following fields when saving"/>
         <TextField fx:id="noWrapFiles" HBox.hgrow="ALWAYS"/>
     </HBox>
-    <RadioButton fx:id="resolveStringsBibTex" text="%Resolve strings for standard BibTeX fields only"
+    <RadioButton fx:id="doNotResolveStrings" text="%Do not resolve BibTeX strings"
                  toggleGroup="$stringsResolveToggleGroup"/>
     <HBox alignment="CENTER_LEFT" spacing="10.0">
-        <RadioButton fx:id="resolveStringsAll" text="%Resolve strings for all fields except"
+        <RadioButton fx:id="resolveStrings" text="%Resolve BibTeX strings for the following fields"
```

Comment:
```suggestion
        <RadioButton fx:id="resolveStringsForFieldsButton" text="%Resolve BibTeX strings for the following fields"
```

Reply:
Hm
The idea was to have an option to disable the resolving completely or activate it for the following fields only. See Screenshot. By default its only activated for the fields 
Otherwise we have the same situation as before

Reply:
This comment was only about the naming of the identifier 😇

Reply:
My driving force was that negations should be avoided in the UI. I was thinking of a checkbox for enabling/disabling the feature. When enabled, one can set which fields... I agree, the current implementation is more consistent to our other properties:

![grafik](https://user-images.githubusercontent.com/1366654/148676570-c79cdfaa-80f1-4322-b6ae-aa38a216fbc0.png)